pkgs/applications/editors/neovim/default.nix

```diff
@@ -108,6 +108,7 @@ let
                 $out/bin/nvim
     '' + optionalString withPython ''
       ln -s ${pythonEnv}/bin/python $out/bin/nvim-python
+      ln -s ${pythonEnv}/bin/python $out/bin/python2
```

Comment:
neovim should not produce `python2` binary. please rename this back to `nvim-python` and wrap neovim binaries with `nvim-python` but expose it as `python` binary, for plugins that look into $PATH for python